.github/workflows/ci_build.yml

```diff
@@ -38,8 +38,8 @@ env:
   DOXYGEN_BRANCH: master
 
   # emscripten
-  EMSCRIPTEN_VERSION: latest
-  EMSCRIPTEN_CACHE_FOLDER: emsdk-cache
+  EM_VERSION: 2.0.7
+  EM_CACHE_FOLDER: 'emsdk-cache'
```

Comment:
Do you think we can try without this first? I quite like using the latest so we are always up-to-date. This is also a good way for us to now straightaway when something breaks with a new version of the emdk. Noticing it straightway usually makes it easier to find where it breaks. (If we need to we can go with this solution then we should not forget to update regularly to a later version.)

Reply:
Yes, I totally agree, and asked about it over there. Meanwhile, I change back to `latest` here